### PR TITLE
iOS 11 bug - popup close button position

### PR DESCRIPTION
**Pull request by iDevelopper**
Fixes a bug in iOS 11 where the system does not take into account the safe area.

### CODE REVIEW

File: LNPopupController/LNPopupController/Private/UIViewController+LNPopupSupportPrivate.m [hunk 1]
```diff
@@ -594,7 +594,7 @@ - (UIEdgeInsets)_vSAIFS
 		}
 		else
 		{
-			return self.popupPresentationContainerViewController.view.superview.safeAreaInsets;
+			return self.popupPresentationContainerViewController.view.safeAreaInsets;
```
Comment (LeoNatan) [inline LNPopupController/LNPopupController/Private/UIViewController+LNPopupSupportPrivate.m:597]: Unfortunately, this creates an issue. In the view controller scene, it miscalculates the safe area of the popup content view controller:

Simulator Screen Shot - iPhone 11 Pro Max (iOS 14 0) - 2020-09-11 at 02 28 39

It takes into account the popup, and it shouldn't. That's why I take the superview's safe area.

**Review by LeoNatan — CHANGES_REQUESTED** [review]

**Comment by LeoNatan** [comment]
Hello, from a quick look, I know there is an issue to use the view directly for safe area calculation.

I will try to see if this can be fixed. Let's keep open for now.

**Comment by LeoNatan** [comment]
OK, I think I found a solution. I will merge, then fix.

**Comment by LeoNatan** [comment]
Thanks!

File: LNPopupController/LNPopupController/Private/UIViewController+LNPopupSupportPrivate.m (hunk 1 above)
Comment (iDevelopper) [reply LNPopupController/LNPopupController/Private/UIViewController+LNPopupSupportPrivate.m:597]: So we could have done like that:
```objective-c
            UIEdgeInsets insets = self.popupPresentationContainerViewController.view.safeAreaInsets;
            UIEdgeInsets popupInsets = _LNPopupSafeAreas(self.popupPresentationContainerViewController);
            UIEdgeInsets finalInsets = UIEdgeInsetsMake(insets.top, insets.left, insets.bottom - popupInsets.bottom, insets.right);
            return finalInsets;
```

**Comment by iDevelopper** [comment]
Thank you ! Happy to join the team of contributors! Despite mistakes...

**Pull request merged**